target/linux/ar71xx/image/generic.mk

```diff
@@ -1,3 +1,20 @@
+define Device/OpenMesh
+  FILESYSTEMS := squashfs
+  DEVICE_PACKAGES := om-watchdog
+  KERNEL := kernel-bin | lzma | uImage lzma
+  IMAGES := sysupgrade.bin factory.bin
+  IMAGE/sysupgrade.bin/squashfs := append-rootfs | pad-rootfs | combined-ext-image
+  IMAGE/factory.bin/squashfs := append-rootfs | pad-rootfs | combined-ext-image
```

Comment:
Drop the factory.bin here. It is identical to the sysupgrade.bin, which means it is waste of build resources on our buildbots and storage resources on our download server.

If someone insists that each board needs a factory.bin in case the sysupgrade.bin can be used for the first install as well, it needs to be fixed in a different way.

Reply:
Oh, I completely missed this mention. Gluon can well deal without a factory image, please remove the factory.bin.